# Planning group and link names
arm_group_name: "panda_arm"
eef_name: "hand"
hand_group_name: "hand"
hand_frame: "panda_link8"

# Poses
hand_open_pose: "open"
hand_close_pose: "close"
arm_home_pose: "ready"

# Scene frames
world_frame: "world"
object_reference_frame: "world"

# Collision object for picking
object_name: "object"
object_types: ["BOX", "CYLINDER", "SPHERE"]
box_dimensions: [0.15, 0.05, 0.05] #[x,y,z]
cylinder_dimensions: [0.1, 0.02] #[height, radius]
sphere_dimensions: [0.025] #[radius]
object_pose: [0.4, 0.3, 0.025, 0.0, 0.0, 0.0]

# Obstacle
obstacle_dimensions: [0.5, 0.2, 0.4] #[x,y,z]
obstacle_pose: [0.45, 0.0, 0.0, 0.0, 0.0, 0.0]

# Gripper grasp frame transform [x,y,z,r,p,y]
grasp_frame_transform: [0, 0, 0.1, 1.571, 0.785, 1.571]

# Place pose [x,y,z,r,p,y]
place_pose: [0.4, -0.3, 0.025, 0, 0, 0]

# Path constraints
path_constrained: [False, True]

# Waypoint density
waypoint_density: [0.02, 0.05, 0.0]
